bigcrunch/webapp.py

```diff
@@ -24,6 +24,8 @@ class ClusterControl(object):
     cluster_name = 'redshift-sqlalchemy-test'
     username = 'travis'
     password = os.environ['PGPASSWORD']
+    vpc_security_group_id = os.environ['VPCSECURITYGROUPID']
+    iam_role = os.environ['IAMROLEARN']
```

Comment:
```suggestion
    vpc_security_group_id = os.environ['VPC_SECURITY_GROUP_ID']
    iam_role = os.environ['IAM_ROLE_ARN']
```

I find these env variable names difficult to read without separators between words, so adding some underscores might make this easier to parse.